Tools/Scripts/build-and-collect-pgo-profiles

```diff
@@ -110,12 +115,12 @@ fi
 
 SPTH='OpenSource/Tools/Scripts'
 
-$SPTH/run-benchmark "${jsargs[@]}"
-$SPTH/pgo-profile merge "$BASE/jetstream"
-
 $SPTH/run-benchmark "${spargs[@]}"
 $SPTH/pgo-profile merge "$BASE/speedometer"
 
+$SPTH/run-benchmark "${jsargs[@]}"
+$SPTH/pgo-profile merge "$BASE/jetstream"
+
```

Comment:
Do we want to swap the order in this change?

Reply:
I did it for testing reasons as Speedometer is faster, I don't think swapping them will create any meaningful changes right?